helix-term/src/args.rs

```diff
@@ -48,6 +49,10 @@ impl Args {
                     Some(path) => args.config_file = Some(path.into()),
                     None => anyhow::bail!("--config must specify a path to read"),
                 },
+                "--logfile" => match argv.next().as_deref() {
```

Comment:
I think I would prefer this as `--log` so it lines up with `--config`. What do you think?